### PR TITLE
fix: saving new hierarchy

### PR DESCRIPTION
**Pull request by tpluscode**
Turns out that I forgot to actually implement the `POST` handler to create a hierarchy

### CODE REVIEW

File: apis/shared-dimensions/lib/domain/hierarchies.ts [hunk 1]
```diff
@@ -10,3 +20,33 @@ export function getHierarchies() {
...
+  }
+
+  const dataset = $rdf.dataset([...resource.dataset].map(replace(resource.term, hierarchyId)))
+  const hierarchy = clownface({ dataset })
+    .namedNode(hierarchyId)
+    .addOut(rdf.type, [hydra.Resource, meta.Hierarchy, md.Hierarchy])
+
+  await store.save(hierarchy)
+  return hierarchy
+}
```
Comment (tpluscode) [inline apis/shared-dimensions/lib/domain/hierarchies.ts:52]: Very similar to how we create shared dimensions

File: apis/shared-dimensions/lib/shapes/hierarchy.ts [hunk 2]
```diff
@@ -111,6 +110,20 @@ export default function (graph: AnyPointer): Initializer<NodeShape> {
...
-    }],
+    }]
+
+    if (rdfTypeProperty) {
+      property.push({
+        path: rdf.type,
+        hasValue: [hydra.Resource, meta.Hierarchy, md.Hierarchy],
+        [dash.hidden.value]: true,
+      })
+    }
```
Comment (tpluscode) [inline apis/shared-dimensions/lib/shapes/hierarchy.ts:121]: The cusp of this change is to add constraint to `rdf:type` property when updating a hierarchy. Not required when creating

**Review by martinmaillard — APPROVED** [review]
Haha, that explains the error I just got

**Comment by l00mi** [comment]
No post not letters ..

**Pull request merged**